config/webpack.config.prod.js

```diff
@@ -0,0 +1,20 @@
+const merge = require('webpack-merge');
```

Comment:
why don't we have source-map here

Reply:
I omitted the dev tool option in here because, by deafult, we want our application to load from bundled code in production.  Another reason is that build will be much faster this way.

Reply:
We need to have source map for production, this helps us debug incase anything funny pops up there

Reply:
Okay. @dinobi I will add the source map.